README.md

```diff
@@ -90,37 +98,37 @@ OK
 ### abe.olin.build/events/
 
 | HTTP Method | Action              |
-| ----------- | ------------------- |
```

Comment:
My code format editor plugin made these changes. If someone else's formatter wants them the other way I can teach it not to so we don't get into dueling commits.